### ISSUE TITLE
ntr: Decreased 12-HETE

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4942]
**Preferred term label:**
Decreased circulating 12-HETE concentration

**Synonyms**
Decreased circulating 12-Hydroxyeicosatetraenoic acid concentration

**Definition (free text, please give PubMed ID)**
A reduction in the concentration of 12-HETE in the blood circulation, a metabolite of arachidonic acid.
Source: CHEBI:19138
need to MIREOT: CHEBI:19138 - 12-HETE

**Parent term (use hpo.jax.org/app)**
HP_0011022 'Abnormal circulating unsaturated fatty acid concentration'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/618372

See https://github.com/monarch-initiative/hpo-annotation-data/issues/387

**Comment by pnrobinson** [comment]
added as HP_0032575
